CHANGELOG.rst

```diff
@@ -1,5 +1,58 @@
 Changelog
 =========
+1.5.0 (2019-12-19)
+-----------------------
+MLflow 1.5.0 includes several major features and improvements:
+
+New Flavors and Flavor Updates:
+
+- New support for a LightGBM flavor (#2136, @harupy)
+- New support for a XGBoost flavor (#2124, @harupy)
+- New support for a Gluon flavor and autologging (#1973, @cosmincatalin)
+- TensorFlow.Keras and Keras parameters are now logged by ``autolog()`` (#2119, @juntai-zheng)
+- Runs automatically created by ``mlflow.tensorflow.autolog()`` and ``mlflow.keras.autolog()`` (#2088) are now automatically ended after training and/or exporting your model. See the `docs <https://mlflow.org/docs/latest/tracking.html#automatic-logging-from-tensorflow-and-keras-experimental>`_ for more details (#2094, @juntai-zheng)
+
+More features and improvements:
+
+- [CLI] When using the ``mlflow server`` CLI command, you can now expose metrics on ``/metrics`` for Prometheus via the optional --activate-parameter argument (#2097, @t-henri)
+- [CLI] The ``mlflow ui`` CLI command now has a ``--host``/``-h`` option to specify user-input IPs to bind to (#2176, @gandroz)
+- [CLI] MLflow now supports pulling git submodules while using MLflow Projects (#2103, @badc0re)
+- [CLI] New ``mlflow models prepare-env`` command to do any preparation necessary to initialize an environment. This allows distinguishing configuration and user errors during predict/serve time (#2040, @aarondav)
+- [API] MLflow ``log_params()`` will recognize Spark ML params as keys and will now extract only the name attribute (#2064, @tomasatdatabricks)
+- [API] Exposes ``mlflow.tracking.is_tracking_uri_set()`` (#2026, @fhoering)
+- [UI] The artifact image viewer nor displays "Loading..." when it is loading an image (#1958, @harupy)
+- [UI] The artifact image view now supports animated GIFs (#2070, @harupy)
+- Adds ability to mount volumes and specify environment variables when using mlflow with docker (#1994, @nlml)
+- Adds run context for detecting job information when using MLflow tracking APIs within Databricks Jobs. The following job types are supported: notebook jobs, Python Task jobs. (#2205, @dbczumar)
+- Improves performance in run requests (#2030, #2059, @jcuquemelle)
```

Comment:
It's kind of weird that some of these have headings like [API], [CLI] etc and some don't. Can you make it consistent?

Reply:
Also it's not clear what "run requests" means here.

Reply:
Clarified to "searching for runs"